workers/java/build.gradle

```diff
@@ -25,6 +25,8 @@ spotless {
     }
 }
 
+compileJava.dependsOn spotlessJava
```

Comment:
This was from this error (local and on CI):

```
Some problems were found with the configuration of task ':compileJava' (type 'JavaCompile').
  - Gradle detected a problem with the following location: '/Users/stephan/Workspace/default/omes/workers/java/build/classes/java/main'.
    
    Reason: Task ':spotlessJava' uses this output of task ':compileJava' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.
```
Not sure why it came up now, though